core-feature-pack/galleon-feature-pack/src/main/resources/layers/standalone/legacy-remoting/layer-spec.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="legacy-remoting">
+    <dependencies>
+        <layer name="application-security-realm"/>
```

Comment:
Hi @darranl , I don't get this point, why is `io` Galleon layer is no longer required on this variant or the remoting feature-group?

Reply:
That is a good point, that probably should be in there.